Resources/doc/custom-id-generators.rts

```diff
@@ -0,0 +1,64 @@
+Custom ID Generators
+====================
+
+Custom ID generators are classes that allow implementing custom logic to generate
+identifiers for your entities. They extend ``Doctrine\ORM\Id\AbstractIdGenerator``
+and implement the custom logic in the ``generate(EntityManager $em, $entity)``
+method. Before Doctrine bundle 2.3, custom ID generators were always created
+without any constructor arguments.
+
+Starting with Doctrine bundle 2.3, the ``CustomIdGenerator`` annotation can be
+used to reference any services tagged with the ``doctrine.id_generator`` tag.
+
+.. code-block:: php
+
+    <?php
+    // User.php
+
+    use Doctrine\ORM\Mapping as ORM;
```

Comment:
Only relevant if this should indeed use the FQCN as service name:

```suggestion
    use App\MyIdGenerator;
    use Doctrine\ORM\Mapping as ORM;
```

Reply:
I removed this example actually now :)

Reply:
So I noticed - I actually liked the example with configuration.

Reply:
I don't think the job of the bundle's doc is to tell how to add a tag, especially when the tag doesn't need attribute and autoconfiguration is good enough (eg for event listeners, the example make sense, because of the attributes)

Reply:
Sounds good 👍